src/Evolution/Executables/Burgers/EvolveBurgers.hpp

```diff
@@ -9,7 +9,6 @@
 #include "Domain/DomainCreators/RegisterDerivedWithCharm.cpp"
```

Comment:
I think this commit needs to be squashed with the second once review is done so each individual commit compiles, right?